code/game/objects/structures/window.dm

```diff
@@ -569,23 +569,23 @@
 	resistance = 6
 
 /obj/structure/window/reinforced/plasma
-	name = "reinforced borosilicate window"
+	name = "reinforced plasma window"
 	desc = "A borosilicate alloy window, with rods supporting it. It seems to be very strong."
 	basestate = "plasmarwindow"
 	icon_state = "plasmarwindow"
 	shardtype = /obj/item/weapon/material/shard/plasma
 	glasstype = /obj/item/stack/material/glass/plasmarglass
-	maximal_heat = T0C + 5227  // Matched to basic.
```

Comment:
We just got done with this, why change it again

Reply:
Currently the only difference reinforced plasma windows have is an altered alpha state to make them look darker. Raising the heat tolerance by five-hundred kelvin would add the material into actual gameplay, as it would be an option for Technomancers or Scientists wishing to shore up a certain section.

Reply:
I did the changes to window temp previously. Don't change the names or anything with the current windows.

Previously I had no reinforced window item to factor in. Add that window with a heat limit of 6000K and significantly more hit points. With this the changes to Toxins Mixing, etc can be done.

Reply:
I'm having a hard time understanding your comment, so help me not have a misunderstanding. Do you want me to not change the file, or are you requesting changes so another project can be done?

Reply:
1. Add a reinforced plasma window spawner.
2. Make the reinforced plasma window resistant to slightly higher temperatures than the basic plasma window.
3. Give the reinforced plasma window more hit points than the basic window.

Reply:
Thank you, It's all clear to me.